addons/l10n_us_account/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'United States - Accounting',
+    'website': 'https://www.odoo.com/documentation/saas-17.2/applications/finance/fiscal_localizations.html',
+    'icon': '/account/static/description/l10n.png',
+    'countries': ['us'],
+    'version': '1.1',
+    'category': 'Accounting/Localizations/Account Charts',
+    'description': """
+    """,
+    'depends': ['l10n_us', 'account'],
+    'installable': True,
+    'auto_install': ['l10n_us', 'account'],
```

Comment:
Shouldn't account be sufficient ?
```suggestion
    'auto_install': ['account'],
```

Reply:
When I do that the template fails as you can see.
I thought that this is not fine that's why I added it.

Reply:
Template failing when doing that :(
I think it's not allowed to have 'l10n_us' in dependencies and not in auto_install

Reply:
> If it is a list, it must contain a **subset of the dependencies**. This module will automatically be installed as soon as all the dependencies in the subset are installed. **The remaining dependencies will be automatically installed as well**. If the list is empty, this module will always be automatically installed regardless of its dependencies and these will be installed as well.

from  https://www.odoo.com/documentation/master/developer/reference/backend/module.html?highlight=auto_install

So it's authorized to have just `account` in the `auto_install`.

So if you have `'auto_install': ['l10n_us', 'account']`, the user still has to manually install `account` and `l10n_us`, for `l10n_us_account` to be auto installed, no ?

Reply:
Have you tried to debug locally the upgrade from 17.2 to master ?

Reply:
Exception has been added by the runbot team for the inconsistent error of the template check!